docs/www/acls.md

```diff
@@ -124,48 +229,121 @@ $ rpk topic describe test-topic \
   0                   1               [1]        [1]               0               
 ```
 
+**_Note_** - If you still have the TLS config from the previous section, you'll also need to pass the TLS flags.
+
+Notice that this command uses Kafka API, so we're using `--brokers <node IP>:<kafka API port>` pattern here.
+
 #### Managing users
 
 While having a superuser is a must to get started, it's not really a good idea to either share the superuser's credentials everywhere or to make everyone a superuser.
 
 You can create, delete and list users with `rpk acl user`.
 
+Remember that since this is a `rpk acl` command the default value of ``--api-urls`` is  `localhost:9644`. If you have a different configuration, you will need to pass `api-urls <node IP>:<admin API port>`.
```

Comment:
Simplify sentence.

```suggestion
The default value of `--api-urls` in the acl command is `localhost:9644`. If you use a different port, pass `api-urls <node IP>:<admin API port>`.
```

Reply:
I really mean configuration here, because if the guy sets a domain for example, he will need to change both node IP and admin port.
But the simplified version is indeed better.

Reply:
I've rewrote this sentence